application/config.json.template

```diff
@@ -95,4 +95,8 @@
         "rateLimitWindowSeconds": 10,
         "rateLimitRequestsInWindow": 3
     }
+    "starboard": {
+        "emojiNames" : ["star"],
+        "channelName": "starboard"
```

Comment:
`channelNamePattern` please (and in the code `Pattern.compile(...)` then, matching the channel)

Reply:
Oh, so in that case I would only have one pattern and not an array right? If the user wants multiple emoji they would need to separate them by a pipe? Is that the design ur referring to?

Reply:
the emoij names can stay non-pattern like that. but the channel name should be a pattern, so that we have some flexibility in naming it.